packages/fxa-shared/l10n/branding.ftl

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### Localization for fxa
+
+## Brand and product terms to be used globally
+
+# Note: Terms should not be transliterated or translated
+# Declension should be avoided
+
+# Firefox and Mozilla Brand
+-brand-name-mozilla = Mozilla
+-brand-name-firefox = Firefox
+
+# Mozilla and Firefox products that should be treated as brands (not translated)
+-product-name-mozilla-vpn = Mozilla VPN
+-product-name-firefox-cloud = Firefox Cloud
```

Comment:
I don't think this was a good choice at the time.

There is no such product as "Firefox Cloud", it's the Cloud Services of Firefox (and only used as a reference in our legal terms).

```
fxa-service-url = { -product-firefox-cloud } Terms of Service
subplat-cloud-terms = { -product-firefox-cloud } Terms of Service
subplat-cloud-terms-plaintext = { subplat-cloud-terms }:
```

This should be either defined as `Firefox Cloud Services` (and update the strings referencing it), or we should just have

```
fxa-service-tos-url = { -brand-name-firefox } Cloud Services Terms of Service
subplat-cloud-terms = { -brand-name-firefox } Cloud Services Terms of Service
subplat-cloud-terms-plaintext = { subplat-cloud-terms }:
```

Reply:
This was the recommendation from legal at the time. I can file another ticket to follow up here.

Reply:
Filed [FXA-6140](https://mozilla-hub.atlassian.net/browse/FXA-6140) to track this.